app/controllers/credence_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CredenceController < ApplicationController
+  before_filter :login_required
+
+  def show
+    @title = "Credence game"
+
+    @game = CredenceGame.find_or_create_by_user_id current_user.id
+    @question = @game.current_question
```

Comment:
This explodes if games haven't been populated yet, and probably should do something more sensible.